gulpfile.js

```diff
@@ -54,15 +59,28 @@ gulp.task('init_module', function(callback) {
   });
 });
 
-gulp.task('build_flash', function(callback) {
-  exec("ant -file build_flash.xml", function(err) {
-    if (err) console.log("Error occured in building osmf plugin : " + err);
-    else osmf_fn();
-    callback();
-  });
+gulp.task('build_flash_osmf', function(callback) {
+    exec("ant -buildfile build_flash.xml -Dbuild=build/classes build-CCswc", function(err) {
+      if (err) console.log("Error occured in building the CC : " + err);
```

Comment:
looks good - i think i would recommend calling the callback either way otherwise you would prevent other modules from building (which are downstream of this task)